toolkit/chrome/global/autocomplete.properties

```diff
@@ -2,6 +2,6 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-searchWithEngine=szukaj w %S
+searchWithEngine=Szukaj w %S
```

Comment:
https://github.com/mozilla/gecko-dev/blame/08997000eb0686614686846a7d6bf97b2ed76fa4/toolkit//content/widgets/autocomplete.xml#L1713-L1721

I think it is used as search engine suggestion in URL bar as "SearchTerm - szukaj w SearchEngine" making this change incorrect.